gg-pac-team3-back/src/routes/userRoutes.mjs

```diff
@@ -0,0 +1,31 @@
+import express from 'express';
+import { getUser, signupUser, getUserDashboard, loginUser, logoffUser, getAllUsers, resetPassword, updateUserProfile } from '../controllers/userController.mjs';
+import { verifyToken } from '../middlewares/verifyToken.mjs';
+
+const router = express.Router();
+
+// Route to get all users records
+router.get('/', verifyToken, getAllUsers);
+
+// Route to get user record by UID
+router.get('/:uid', verifyToken, getUser);
+
+// Route for user dashboard
+router.get('/dashboard', verifyToken, getUserDashboard);
+
+// Route for user signup (Creating a new user)
+router.post('/', signupUser);
+
+// Route for user login
+router.post('/login', loginUser);
+
+// Route for user logoff
+router.delete('/login', verifyToken, logoffUser);
```

Comment:
This is a very atypical pattern - to use a DELETE request to log a user out. You're not really deleting anything on the server. The more conventional would be 
`router.post('/logout', verifyToken, logoffUser);`

Reply:
okay, I will make sure to do that.